commands/preview-group/helpers/codeReader.cs

```diff
@@ -139,7 +142,7 @@ public static (ushort?, ushort?) GetLineNumbers(string fragment)
             fragment = fragment.Replace("L", "");
             ushort? startLine = null;
             ushort? endLine = null;
-            if (fragment.Contains('-'))
+            if (fragment.Contains('-') && fragment.IndexOf('-', StringComparison.Ordical) > 0)
```

Comment:
This does not compile. because of a typo `StringComparison.Ordical` should be `StringComparison.Ordinal`